tests/system/action/resource/test_upload.py

```diff
@@ -171,7 +171,7 @@ def test_upload_and_mixed_sep_actions(self) -> None:
 
         self.assert_status_code(response, 400)
         self.assertIn(
-            "Datastore service sends HTTP 400. Model 'resource/1' raises MODEL_LOCKED error.",
+            "Datastore service sends HTTP 400. The following locks were broken: 'resource/1', 'organization/1/resource_ids', 'organization/1/resource_ids', 'resource/1/organization_id', 'resource/organization_id', 'resource/token'",
```

Comment:
Can't you prevent fields appear twice, like 'organization/1/resource_ids'?

Reply:
Yes, it is in work :D see https://github.com/OpenSlides/openslides-datastore-service/pull/129

I did not initially notice this, I just copy-pasted the errors from the tests. I will update this PR once https://github.com/OpenSlides/openslides-datastore-service/pull/129 is merged.

Reply:
done.